article/sections/_1_intro_generic.tex

```diff
@@ -27,8 +27,8 @@ \section{Introduction}
 incorporating causal inference techniques where appropriate and from contributing
 to the causal inference literature where it's lacking.
 
-The relevance of this topic now is motivated by the significant boost in the causal inference literature recently, both in the potential outcomes and the causal graphical modeling frameworks.
-The goal of this chapter is to formalize a workflow for approaching 
+This topic is even more relevant now because of the significant and recent boost in the causal inference literature recently, both in the potential outcomes and the causal graphical modeling frameworks.
```

Comment:
```suggestion
This topic is even more relevant now because of the significant and recent boost in the causal inference literature, both in the potential outcomes and the causal graphical modeling frameworks.
```